_docs/info/migrate_rpool.md

```diff
@@ -0,0 +1,122 @@
+
+So after completing this guide your rpool should look like this:
+
+``` terminal
+$ zpool status
+  pool: rpool
+ state: ONLINE
+  scan: resilvered 1.59G in 0h11m with 0 errors on Sun Apr 12 00:35:53 2020
+config:
+
+        NAME        STATE     READ WRITE CKSUM
+        rpool       ONLINE       0     0     0
+          mirror-0  ONLINE       0     0     0
+            c1t3d0  ONLINE       0     0     0
+            c1t4d0  ONLINE       0     0     0
+
+```
+
+## Attach the new disk(s) to rpool
+
+The first step is to attach the new disks to the existing rpool using the command **zpool attach** like this:
+
+``` terminal
+$ sudo zpool attach rpool c1t1d0 c1t3d0
+Password: 
+Make sure to wait until resilver is done before rebooting.
+```
+
+**You must not do anything until resilver is done**. So go and grab a cup of coffee or some other fluid you want to drink.
+
+To monitor progress you can use this command:
+
+``` terminal
+$ yes "zpool status; sleep 10"|sh
+
+Use Ctrl+c to quit the command.
+```
+
+The above should be done for every new disk which in this case means one more time for c1t4d0
+
+## Install boot block and boot loader to the new disk(s)
+
+The next step is to install the boot block and the boot loader to the new disk(s) using the command **bootadm install-bootloader** like this:
+
+``` terminal
+$ sudo bootadm install-bootloader -P rpool
+```
+
+## Reboot to test migration was successful
+
+Before detaching the old disk(s) from the rpool we make a reboot to ensure everything was successful. **Remember to change the first boot device in the BIOS to use the new disk (or one of the new disk(s)).**
+
+When rebooting you might see a warning from 'devid register' that device ids has changed in case you need to reorder the disks in the BIOS to be able to change boot device. This is just a warning which will be fixed by creating a new boot_archive either manually or automatically by OmniOSce when you reboot or power down your system.
+
+## Detach old disk(s) from the rpool
+
+If you were able to make a successful reboot on the new disk(s) then the time has come to detach the old disk(s) from the rpool using the command **zpool detach** like this:
+
+``` terminal
+sudo zpool detach rpool c1t1d0
+sudo zpool detach rpool c1t2d0
+```
+
+After detaching the old disk(s) from the rpool you should do a reboot to ensure everything is fine.
+
+When rebooting you might see a warning from 'devid register' that device ids has changed when removing the old disk(s). This is just a warning which will be fixed by creating a new boot_archive either manually or automatically by OmniOSce when you reboot or power down your system.
+
+## Contact
+
+If you have any problems or questions, please get in touch via
+[the lobby](https://gitter.im/omniosorg/Lobby) or
+[#omnios on Freenode](http://webchat.freenode.net?randomnick=1&channels=%23omnios&uio=d4)
+
```

Comment:
Can you replace this with:

```
Any problems or questions, please [get in touch](/about/contact.html).
```

which takes them to the contact page and centralises the various contact methods.
Thanks